tests/network/services.go

```diff
@@ -170,6 +171,101 @@ var _ = SIGDescribe("Services", func() {
 			})
 		})
 	})
+
+	Context("Masquerade interface binding", func() {
+		var inboundVMI *v1.VirtualMachineInstance
+		var serviceName string
+
+		const (
+			selectorLabelKey   = "expose"
+			selectorLabelValue = "me"
+			servicePort        = 1500
+		)
+
+		createReadyVMIWithMasqueradeBindingAndExposedService := func(hostname string, subdomain string) *v1.VirtualMachineInstance {
+			vmi := libvmi.NewCirros(
+				libvmi.WithInterface(libvmi.InterfaceDeviceWithMasqueradeBinding()),
+				libvmi.WithNetwork(v1.DefaultPodNetwork()))
+			return readyVMI(
+				exposeExistingVMISpec(vmi, subdomain, hostname, selectorLabelKey, selectorLabelValue))
+		}
+
+		BeforeEach(func() {
+			subdomain := "my-subdomain"
+			hostname := "myvmi"
+
+			inboundVMI = createReadyVMIWithMasqueradeBindingAndExposedService(hostname, subdomain)
+
+			tests.StartTCPServer(inboundVMI, servicePort)
+		})
+
+		Context("with a service matching the vmi exposed", func() {
+			var isDualStack bool
+			var ipv6ServiceName string
+			var ipv6ServicePort int
+
+			BeforeEach(func() {
+				serviceName = "myservice"
+
+				service := buildServiceSpec(serviceName, servicePort, servicePort, selectorLabelKey, selectorLabelValue)
+				_, err := virtClient.CoreV1().Services(inboundVMI.Namespace).Create(service)
+				Expect(err).ToNot(HaveOccurred())
+
+				isDualStack = flags.IsDualStackedNetworkConfig
+				if isDualStack {
+					ipv6ServiceName = serviceName + "v6"
+					ipv6ServicePort = servicePort + 6
```

Comment:
We cannot expose the same port one for IPv4 and one for IPv6?

Reply:
> I forgot about a known issue we have with the jobs (which have been changed from simple pods to controler jobs).
> We assumed these are deleted with the TTL options, but they are not (it is feature gated).
> 
> We need to delete the jobs, like it is done here: https://github.com/kubevirt/kubevirt/pull/3871/files#diff-f911024a391a99cfdef2241f409a4748R850
> 
> Previously this has been mitigated by cleaning the namespace from time to time, but now we do not have it here so this may be an issue.
> Said that, we can fix it in a following PR, up to you.

Thanks for raising it; I had noticed it when running locally, but I am not used to the namespace being kept in the tests this folder, and as such, didn't take it into account.

I think it's only fair to do it in this PR; shouldn't take that much more effort.